scripts/gen_manifests.sh

```diff
@@ -11,6 +16,38 @@ else
 fi
 
 cd "${WORKFLOW_ID}" || exit
+
+echo -e "\nquarkus.flyway.migrate-at-start=true" >> application.properties
+
 # TODO Update to use --skip-namespace when the following is released
 # https://github.com/apache/incubator-kie-tools/pull/2136
 ../kn workflow gen-manifest --namespace ""
+
+
+if [ "$ENABLE_PERSISTENCE" = false ]; then
+  exit
+fi
+
+yq --inplace eval '.metadata.annotations["sonataflow.org/profile"] = "prod"' manifests/01-sonataflow*.yaml
+
+yq --inplace ".spec.podTemplate.container.image=\"quay.io/orchestrator/serverless-workflow-${WORKFLOW_ID}:latest\"" manifests/01-sonataflow*.yaml
```

Comment:
Maybe instead of `manifests/01-sonataflow*.yaml` we could use `manifests/01-sonataflow-${WORKFLOW_ID}.yaml`

Reply:
should be `manifests/01-sonataflow_${WORKFLOW_ID}.yaml`
(underscore instead of dash before the workflow ID).

Reply:
I replaced this with a variable instead. makes more sense.